src/data/taskList.ts

```diff
@@ -34,11 +37,31 @@ export const businessCaseTag = (intakeStatus: string) => {
     case 'BIZ_CASE_DRAFT_SUBMITTED':
     case 'BIZ_CASE_FINAL_NEEDED':
     case 'BIZ_CASE_FINAL_SUBMITTED':
+    case 'LCID_ISSUED':
       return 'COMPLETED';
     case 'NOT_IT_REQUEST':
-    case 'LCID_ISSUED':
       return 'NOT_NEEDED';
     default:
       return '';
   }
 };
+
+// Task List Item: Attend GRB Meeting
+export const attendGrbMeetingTag = (intake: SystemIntakeForm) => {
+  if (intake.requestType === 'RECOMPETE') {
+    return 'NOT_NEEDED';
+  }
+
+  switch (intake.status) {
+    case 'READY_FOR_GRB':
+      return '';
```

Comment:
Is there a benefit to returning the original value here, and checking that on line 122 of the TaskListCta, instead of the intake status directly? I don't feel very strongly about this, but it would make the code more consistent between different steps. In code:

```
case 'READY_FOR_GRB':
      return 'READY_FOR_GRB';
```
and then 
```
if (attendGrbMeetingTag(intake) === 'READY_FOR_GRB') {
    return (
      <UswdsLink
        data-testid="prepare-for-grb-btn"
        className="usa-button"
        variant="unstyled"
        asCustom={Link}
        to={`/governance-task-list/${intake.id}/prepare-for-grb`}
      >
        Prepare for the Review Board meeting
      </UswdsLink>
    );
  }
```

Reply:
Those functions in `data/taskList.ts` are specifically for the "tag" or "status" labels on the task list items. They should only return `COMPLETED`, `CANNOT_START`, `NOT_NEEDED`, or `''` (empty string represents nothing). I'll add a static type to represent the return value.

However, to address the consistency, we'd probably want to change it to:
```
  if (['LCID_ISSUED', 'WITHDRAWN', 'NOT_IT_REQUEST', 'NOT_APPROVED', 'NOT_GOVERNANCE'].includes(intake.status)) {
    return (
      <UswdsLink
        data-testid="prepare-for-grb-link"
        asCustom={Link}
        to={`/governance-task-list/${intake.id}/prepare-for-grb`}
      >
        Prepare for the Review Board meeting
      </UswdsLink>
    );
  }
```

Let me think on it.